sky/shell/platform/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -0,0 +1,57 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "sky/shell/platform/ios/framework/Headers/FlutterAppDelegate.h"
+#include "sky/shell/platform/ios/framework/Headers/FlutterViewController.h"
+
+@implementation FlutterAppDelegate {
+  UIWindow* _window;
```

Comment:
Ah, ok.  Maybe I should just move it back to the header the way you had it before.